meson.build

```diff
@@ -521,10 +521,11 @@ foreach ident : [
                                  #include <unistd.h>'''],
         ['bpf',               '''#include <sys/syscall.h>
                                  #include <unistd.h>'''],
-        ['explicit_bzero' ,   '''#include <string.h>'''],
         ['statx',             '''#include <sys/types.h>
                                  #include <sys/stat.h>
                                  #include <unistd.h>'''],
+        ['explicit_bzero' ,   '''#include <string.h>'''],
+        ['reallocarray',      '''#include <malloc.h>'''],
```

Comment:
not that it matters, but i figure sooner or later we should sort these alphabetically...

Reply:
Function calls have stubs in missing_syscall.h, and glibc functions get a replacement somewhere in basic/. So I think it's good not to sort them all alphabetically, but to keep the order in missing_syscall.h and here the same.